src/gui/guirequests.h

```diff
@@ -123,6 +123,39 @@ struct GuiRequests {
         static ExitCode setLaunchOnStartup(bool enabled);
         static ExitCode getAppState(AppStateKey key, QString &value);
         static ExitCode updateAppState(AppStateKey key, const QString &value);
+        template <typename T>
```

Comment:
I would prefer using a std::variant

Reply:
I changed the implementation to use std::variant in commit: [1885599](https://github.com/Infomaniak/desktop-kDrive/pull/117/commits/18855991e78e0dd15df6c38db404f2a38f5c85eb)